configuration/pih/htmlforms/mentalHealth.xml

```diff
@@ -793,7 +793,76 @@
                 </p>
 
             </div>
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.TraumaticEvent"/>
+                    </label>
+                    <obs conceptId="PIH:12362" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
+                </p>
+
+            </div>
+        </div>
+
+        <div class="section-container" id="radio_button_container">
+
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.PoorIntrospection"/>
+                    </label>
+                    <obs conceptId="CIEL:166051" style="radio" answerConceptIds="PIH:YES,PIH:NO" answerCodes="yes,no" answerSeparator=""/>
+                </p>
+
+            </div>
+            <div class="col">
+                <p>
+                    <label>
+                        <uimessage code="pihcore.mentalhealth.ThoughtContent"/>
```

Comment:
Your label is "Thought content" but the obs/question is "Muscle tone and strength"